neo/Network/P2P/Payloads/BlockBase.cs

```diff
@@ -50,8 +50,9 @@ Witness[] IVerifiable.Witnesses
         public virtual void Deserialize(BinaryReader reader)
         {
             ((IVerifiable)this).DeserializeUnsigned(reader);
-            if (reader.ReadByte() != 1) throw new FormatException();
-            Witness = reader.ReadSerializable<Witness>();
+            Witness[] witnesses = reader.ReadSerializableArray<Witness>();
+            if (witnesses.Length != 1) throw new FormatException();
+            Witness = witnesses[0];
```

Comment:
I think we should have a single witness here, using getters and setters, but I'll leave this to another PR.

Reply:
Is not the same

Reply:
This is not the same? How?

Reply:
This byte specify that is a block, 0 is a header, is not an array

https://github.com/neo-project/neo/blob/master/neo/Network/P2P/Payloads/Header.cs#L37

Reply:
We could use it as array, but is for specify the type i think

Reply:
But I'll propose that in another PR.